includes/fbbot/class-omise-fbbot-user-service.php

```diff
@@ -0,0 +1,86 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+if ( class_exists( 'Omise_FBBot_User_Service' ) ) {
+  return;
+}
+
+class Omise_FBBot_User_Service {		
+	private function __construct() {
+		// Hide the constructor
+	}
+
+	public static function get_user( $sender_id ) {
+		$user = get_user_by( 'login', $sender_id );
+
+		if ($user) {
+			return $user->data;
+		}
+
+		$fb_profile = self::get_fb_profile( $sender_id );
+		if ( ! $fb_profile ) 
+			return NULL;
+
+		$wp_user = self::create_wp_user( $fb_profile );
```

Comment:
@igroomgrim ✋ Do we really need to immediately create a new user when call `get_user`?
Saw that user object is being used to just get `display_name` to show on order transaction and greeting message.

Note. For order, it could be ordered by a 'guest' user.